config/optional/block.block.oe_content_language_switcher.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_multilingual
+  theme:
+    - oe_theme
+id: oe_content_language_switcher
+theme: oe_theme
+region: content
```

Comment:
By looking at the [specifications of the ECL component](https://ec-europa.github.io/europa-component-library/ec/components/detail/ec-component-lang-select-page--default) this component should go under the breadcrumbs. So it should be in the breadcrumbs region. Note that also the primary color should be fixed.

Reply:
The problem with this is that the current ecl implementation has a background color so it doesn't much up at all if placed in the breadcrumbs or navigation regions. The only place where it doesn't stick out like a sore thumb is on the content region. I propose we solve this in a follow up ticket